tests/test_socket_server.py

```diff
@@ -97,6 +101,8 @@ async def test_full_command_sequence(self):
         await self.write(command="start", parameters={})
         self.data = await self.read()
         self.assertEqual(ResponseCode.OK, self.data["response"])
+        self.data = await self.read()
```

Comment:
Is there anything more you can test about the data, beyond the response code?

Reply:
Done.